include/libvfio-user.h

```diff
@@ -240,6 +245,26 @@ typedef ssize_t (vfu_region_access_cb_t)(vfu_ctx_t *vfu_ctx, char *buf,
  *    and writes are an error
  *  - otherwise, the callback is expected to handle the access
  *
+ * Regions VFU_PCI_DEV_MIGR_REGION_IDX and VFU_GENERIC_DEV_MIGR_REG_IDX,
+ * corresponding to the migration region, enable live migration support for
+ * the device. The migration region must contain at the beginning the migration
+ * registers (struct vfio_device_migration_info defined in <linux/vfio.h>) and
+ * the remaining part of the region can be arbitrarily used by the device
+ * implementation. Therefore, the size of the migration region must be at least
+ * sizeof(struct vfio_device_migration_info). libvfio-user offers two ways for
```

Comment:
Can we clarify this, specifically say something like:

The region provided must have at least vfu_get_migr_regs_size() bytes available at the start of the region (this size is guaranteed to be page-aligned). If mmap_areas is given, it must *not* include this part of the region.

Reply:
Sure, I was hoping for a better wording :).